appknox/client.py

```diff
@@ -144,16 +144,13 @@ def file_get(self, file_id):
         url = 'files/' + str(file_id)
         return self._request(requests.get, url)
 
-    def file_list(self, project_id):
+    def file_list(self, project_id, limit, offset):
```

Comment:
@darshanime Default params please. Declare a variable and use that in both the default places.